### DISCUSSION TITLE
怎样彻底关闭代码分割splitChunks？

### DISCUSSION THREAD

**Discussion by 006hjy** [vuepress-theme-hope/vuepress-theme-hope#3756] [Q&A] [answered]
编译出的页面需要给第三方站点使用，不知道在哪里关闭splitChunks。

**Comment by Mister-Hope** [comment]
你的代码分割指的是什么？

**Reply by 006hjy** [reply]
webpack的splitChunks

**Reply by Mister-Hope** [reply]
那跟代码分割无关，自行覆盖主题组件以控制各部分使用 a 标签直接跳转主页。

**Reply by 006hjy** [reply]
目的是要禁用掉所有页面对主页的预加载。因为主页是第三方页面。如果不禁用掉的话，从其他页面点击图标返回主页，显示的是错误的主页。必须要手动刷新一下，才会显示回正确的主页。

**Reply by 006hjy** [reply]
> 那跟代码分割无关，自行覆盖主题组件以控制各部分使用 a 标签直接跳转主页。

navbar顶栏左上角的网站标题，怎么改点击跳转？

**Reply by Mister-Hope** [reply, marked as answer]
把对应的组件覆盖一下。
